src/main/java/eu/dissco/core/translator/terms/specimen/TypeStatus.java

```diff
@@ -13,15 +13,12 @@ public class TypeStatus extends Term {
   private final List<String> dwcaTerms = List.of(TERM);
```

Comment:
This is still a bit open to disucssion. However we will try to find matching terms within existing ontologies as much as possible with dwc being the first choise and abcd(efg) second. As there seems to be an exact match with the DWC term I think we should use that one. We need to get conformation from the community however, not sure when this will happen.